.circleci/config.yml

```diff
@@ -115,7 +115,7 @@ references:
     name: Install npm dependencies
     command: |
       if [ ! -d node_modules ]; then
-        npm ci
+        npx lerna bootstrap
```

Comment:
this is magic. lerna notices we're in circle and actually runs `npm ci` under the hood